backend/app/api/endpoints/deliveries.py

```
@@ -255,15 +258,43 @@ def list_loop_items(
     response_model=LoopItemResponse,
     status_code=status.HTTP_201_CREATED,
 )
-def create_loop_item(
+async def create_loop_item(
     project_id: int,
     values: LoopItemCreate,
     db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)